tests/phpunit/tests/feed/atom.php

```diff
@@ -81,6 +94,24 @@ private function do_atom() {
 		return $out;
 	}
 
+	/**
+	 * This is a bit of a hack used to buffer feed content.
+	 */
+	private function do_atom_comments() {
+		ob_start();
+		// Nasty hack! In the future it would better to leverage do_feed( 'atom' ).
```

Comment:
As this ticket has been marked for a future release, it would be good to have this idea plan to leverage `do_feed( 'atom' )` followed up.